src/commands/logStream/RevisionListStep.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import type { ContainerAppsAPIClient, Revision } from "@azure/arm-appcontainers";
+import { uiUtils } from "@microsoft/vscode-azext-azureutils";
+import { AzureWizardPromptStep, IAzureQuickPickItem, createSubscriptionContext, nonNullProp } from "@microsoft/vscode-azext-utils";
+import * as dayjs from 'dayjs';
+import { createContainerAppsAPIClient } from "../../utils/azureClients";
+import { localize } from "../../utils/localize";
+import { IStreamLogsContext } from "./IStreamLogsContext";
+
+export class RevisionListStep extends AzureWizardPromptStep<IStreamLogsContext> {
+    public async prompt(context: IStreamLogsContext): Promise<void> {
+        if (context.containerApp.revisionsMode === 'Multiple') {
+            const placeHolder: string = localize('selectRevision', 'Select a revision');
+            context.revision = (await context.ui.showQuickPick(this.getPicks(context), { placeHolder })).data;
+        } else {
+            (await this.getPicks(context)).forEach(revision => {
```

Comment:
You can do this to get the latest/active revision. Single revision's latest one should always be the active, AFAIK.

`const revisionData = await client.containerAppsRevisions.getRevision(this.resourceGroup, this.name, nonNullProp(this.containerApp, 'latestRevisionName'));`